lib/soc/interrupt_guard.hpp

```diff
@@ -0,0 +1,46 @@
+#pragma once
+
+/*
+    Name: interrupt_guard.hpp
+
+    Copyright(c) 2020 Jay Kickliter
+    This code is licensed under MIT license (see LICENSE file for details)
+*/
+
+#include <cstdint>
+
+namespace soc {
+    /**
```

Comment:
Remove indent for whole `soc` namespace

Reply:
Sorry. This is why I always check in a `.clang-format` file and run `git clang-format` in CI.

Reply:
Ok, I'll prepare clang format config for whole project